Update Radio content (#941)
hashicorp/design-system: website/docs/components/form/radio/partials/code/how-to-use.md

</Hds::Form::Radio::Group>
```

#### Extra content in legend and helper text

> The `Legend` and `HelperText` contextual components used in the "group" are yielding their content: this means you can pass not just plain text, but also structured content.

Reviewer: ```suggestion
The `Legend` and `HelperText` contextual components used in the "group" yield their content: meaning you can pass not just plain text, but also structured content.
```

</Hds::Form::Radio::Group>
```

#### Validation

> To show the user that their input is not valid, you have to provide an error message (using the `Error` contextual component).

Reviewer: ```suggestion
If an input is not valid, provide the user with an error message using the `Error` contextual component.
```
